assets/js/modules/search-console/components/widgets/PopularKeywordsWidget.stories.js

```diff
@@ -150,12 +189,26 @@ export default {
 					},
 				] );
 
-				registry.dispatch( CORE_USER ).setReferenceDate( '2020-09-08' );
+				provideModuleRegistrations( registry );
+
+				const [ accountID, propertyID, webDataStreamID ] = [
+					'12345',
+					'34567',
+					'56789',
+				];
+
+				// Set accountID, propertyID and webDataStreamID values.
 				registry
-					.dispatch( MODULES_SEARCH_CONSOLE )
-					.receiveGetSettings( {
-						propertyID: 'http://example.com/',
-					} );
+					.dispatch( MODULES_ANALYTICS )
+					.setAccountID( accountID );
+				registry
+					.dispatch( MODULES_ANALYTICS_4 )
+					.setPropertyID( propertyID );
+				registry
+					.dispatch( MODULES_ANALYTICS_4 )
+					.setWebDataStreamID( webDataStreamID );
```

Comment:
We don't need to mock the Analytics data since this tile's report depends on the Search Console module. We could set the SC property ID according to the `getServiceEntityAccessURL` selector. Even that's unnecessary as the selector doesn't return `undefined`. However, it's preferred.